ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -7,21 +7,10 @@ import SwiftUI
 
 public final class AliasTokens {
 
-    // MARK: ForegroundColors
+    // MARK: Colors
```

Comment:
nit: I think the git diff would be easier to read if we move colors to the bottom, so it's more clear that all the new values were removed and put in one group.